src/compiler/checker/lines_18000_20000.rs

```diff
@@ -261,7 +262,7 @@ impl<'type_checker> CheckTypeRelatedTo<'type_checker> {
             if self.should_check_as_excess_property(prop.clone(), source.symbol()) && true {
                 if !self.type_checker.is_known_property(
                     reduced_target.clone(),
-                    &prop.escaped_name,
+                    prop.escaped_name(),
```

Comment:
Since `Symbol` became an enum it now uses `SymbolInterface` methods everywhere to access/modify "fields"